PCbuild/build.bat

```diff
@@ -73,6 +75,8 @@ if "%~1"=="-d" (set conf=Debug) & shift & goto CheckOpts
 if "%~1"=="-m" (set parallel=/m) & shift & goto CheckOpts
 if "%~1"=="-M" (set parallel=) & shift & goto CheckOpts
 if "%~1"=="-v" (set verbose=/v:n) & shift & goto CheckOpts
+if "%~1"=="-vv" (set verbose=/v:d) & shift & goto CheckOpts
+if "%~1"=="-q" (set verbose=/v:q) & shift & goto CheckOpts
```

Comment:
Does this one also need `/nologo` and perhaps `/clp:summary` for the case where there are errors? (See the default value at line 63 post-change.)

Reply:
Good idea, I've set `-vv` to use `/ds` (detailed summary) and quiet to use console logger summary and nologo